volatility3/framework/plugins/linux/mountinfo.py

```diff
@@ -115,8 +86,8 @@ def get_mountinfo(
         if not mnt_root:
             return None
 
-        path_root = cls._do_get_path(mnt, task.fs.root)
-        if path_root is None:
+        path_root = linux.LinuxUtilities._get_path_mnt(task, mnt)
```

Comment:
Sorry, just noticed, this is a bad.  5:P  Again, it's a call to a private member, so probably this needs the `_` removing from the front of the function name, and then a `VersionRequirement` adding to plugins like this that call it, on the version that contains it.  Sorry I missed this earlier...